chemclipse/plugins/org.eclipse.chemclipse.swt.ui/src/org/eclipse/chemclipse/swt/ui/components/SearchSupportUI.java

```diff
@@ -152,18 +152,20 @@ public void widgetDefaultSelected(SelectionEvent e) {
 		return text;
 	}
 
-	private Button createCheckBoxCaseSensitive(Composite parent) {
+	private Button createButtonCaseSensitive(Composite parent) {
 
-		Button button = new Button(parent, SWT.CHECK);
+		Button button = new Button(parent, SWT.PUSH);
 		button.setText("");
 		button.setToolTipText("Search Case Sensitive");
-		button.setSelection(PreferenceSupplier.isSearchCaseSensitive());
+		button.setImage(ApplicationImageFactory.getInstance().getImage(IApplicationImage.IMAGE_CASE_SENSITIVE, IApplicationImage.SIZE_16x16));
```

Comment:
@Mailaender When initializing the button image, it should be checked if a case sensitive search shall be performed. Also use the decorator here:

`button.setImage(ApplicationImageFactory.getInstance().getImage(IApplicationImage.IMAGE_CASE_SENSITIVE, IApplicationImage.SIZE_16x16, PreferenceSupplier.isSearchCaseSensitive()));`

Reply:
Added the missing initialization.

Reply:
I had to revert this due to performance problems.